### ISSUE TITLE
Accessibility Issue: Skroutz Analytics Injected <iframe> Missing title Attribute

### ISSUE THREAD

**Issue by KonstantinosVazaios** [skroutz/developer.skroutz.gr#261]
We're using Skroutz Analytics on our ReactJS Single Page Application via the official tracking script. The script injects an <iframe> into the DOM like this:

<iframe name="easyXDM_default2525_provider" id="easyXDM_default2525_provider" src="https://skroutza.skroutz.gr/track/connect?shop_code=SA-15....>

However, this iframe does not include a title attribute, which violates accessibility standards. As a result, AChecker Accessibility score is no longer 100%.

Violation: <iframe> elements must have a unique, non-empty title attribute that describes the embedded content.

It's important to mention that I cannot find myself the iframe that is being generated if I try to inspect html.

![Image](https://github.com/user-attachments/assets/d6322a13-2cd9-461f-854b-165832f54d52)
